composer.json

```diff
@@ -15,19 +15,18 @@
         }
     ],
     "require": {
-        "php": ">5.4",
-        "paragonie/random_compat": "~1.0|~2.0|~9.99.99",
-        "symfony/framework-bundle": "~2.3|~v3.0|~4.0|~5.0|~6.0",
-        "symfony/security-core": "~2.3|~3.0|~4.0|~5.0|~6.0",
-        "symfony/security-http": "~2.3|~3.0|~4.0|~5.0|~6.0",
-        "symfony/security-csrf": "~2.3|~3.0|~4.0|~5.0|~6.0",
-        "symfony/yaml": "~2.3|~3.0|~4.0|~5.0|~6.0",
+        "php": "^7.4|^8.0",
+        "symfony/framework-bundle": "~4.4|~5.0|~6.0",
+        "symfony/security-core": "~4.4|~5.0|~6.0",
+        "symfony/security-http": "~4.4|~5.0|~6.0",
+        "symfony/security-csrf": "~4.4|~5.0|~6.0",
+        "symfony/yaml": "~4.4|~5.0|~6.0",
```

Comment:
```suggestion
        "symfony/framework-bundle": "~4.4|~5.0|~6.0",
        "symfony/security-core": "~4.4|~5.4|~6.0",
        "symfony/security-http": "~4.4|~5.4|~6.0",
        "symfony/security-csrf": "~4.4|~5.4|~6.0",
        "symfony/yaml": "~4.4|~5.4|~6.0",
```
Let's only allow supported versions (right now 5.3 is supported, but by the time we release a new major version it won't)